### PR TITLE
Test setting S3 region explicitly

### PR DESCRIPTION
**Pull request by skrzypo987**
More of an end-to-and test of https://github.com/trinodb/trino/pull/14398

## Non-technical explanation
Just a new test

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by kokosing — APPROVED** [review]

File: plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java [hunk 1]
```diff
@@ -42,6 +42,12 @@
 {
     public static final String MINIO_ACCESS_KEY = "accesskey";
     public static final String MINIO_SECRET_KEY = "secretkey";
+    /**
+     * In S3 this region is implicitly the default one. In Minio, however,
+     * if we set an empty region, it will accept any.
+     * So setting it by default to `us-east-1` simulates S3 better
+     */
+    public static final String MINIO_DEFAULT_REGION = "us-east-1";
```
Comment (Praveen2112) [inline plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java:50]: Is it possible to change it to a non default region ? With the default region set to `us-east-1`. Will the test pass if `hive.s3.region` is not configured ?

Comment (skrzypo987) [reply plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java:50]: If we change it here all tests will fail. Unless we change it to empty.
`hive.s3.region` being null will result in `us-east-1` being implicitly selected

**Review by Praveen2112 — APPROVED** [review]

**Comment by skrzypo987** [comment]
@kokosing @Praveen2112 I see that there are no more remarks and this is approved. Can we get this merged?

**Review by kokosing — APPROVED** [review]

**Comment by kokosing** [comment]
Thanks 👍

**Pull request merged**